2025-02-24 08:12:39,646 INFO    MainThread:42048 [wandb_setup.py:_flush():67] Current SDK version is 0.19.7
2025-02-24 08:12:39,646 INFO    MainThread:42048 [wandb_setup.py:_flush():67] Configure stats pid to 42048
2025-02-24 08:12:39,646 INFO    MainThread:42048 [wandb_setup.py:_flush():67] Loading settings from /Users/namle/.config/wandb/settings
2025-02-24 08:12:39,646 INFO    MainThread:42048 [wandb_setup.py:_flush():67] Loading settings from /Users/namle/DATN/Sniffer/wandb/settings
2025-02-24 08:12:39,646 INFO    MainThread:42048 [wandb_setup.py:_flush():67] Loading settings from environment variables
2025-02-24 08:12:39,646 INFO    MainThread:42048 [wandb_init.py:setup_run_log_directory():647] Logging user logs to /Users/namle/DATN/Sniffer/wandb/offline-run-20250224_081239-zxf1s13f/logs/debug.log
2025-02-24 08:12:39,646 INFO    MainThread:42048 [wandb_init.py:setup_run_log_directory():648] Logging internal logs to /Users/namle/DATN/Sniffer/wandb/offline-run-20250224_081239-zxf1s13f/logs/debug-internal.log
2025-02-24 08:12:39,646 INFO    MainThread:42048 [wandb_init.py:init():761] calling init triggers
2025-02-24 08:12:39,646 INFO    MainThread:42048 [wandb_init.py:init():766] wandb.init called with sweep_config: {}
config: {'config': {'run': {'task': 'captioning', 'lr_sched': 'linear_warmup_cosine_lr', 'init_lr': 1e-05, 'min_lr': 0, 'warmup_lr': 1e-08, 'warmup_steps': 1000, 'weight_decay': 0.05, 'max_epoch': 10, 'batch_size_train': 1, 'batch_size_eval': 1, 'num_workers': 4, 'accum_grad_iters': 1, 'max_len': 256, 'min_len': 8, 'num_beams': 1, 'seed': 42, 'output_dir': 'output/blip2_vicuna_instruct/ooc_7b', 'amp': True, 'resume_ckpt_path': None, 'evaluate': False, 'train_splits': ['train'], 'valid_splits': ['val'], 'test_splits': ['test'], 'device': 'cuda', 'world_size': 1, 'dist_url': 'env://', 'distributed': False}, 'model': {'arch': 'blip2_vicuna_instruct', 'load_finetuned': False, 'load_pretrained': True, 'pretrained': 'llm-ckpt/instruct_blip_vicuna7b_trimmed.pth', 'finetuned': '', 'image_size': 224, 'drop_path_rate': 0, 'use_grad_checkpoint': True, 'vit_precision': 'fp16', 'freeze_vit': True, 'num_query_token': 32, 'llm_model': 'llm-ckpt/vicuna-7b-v1.1', 'prompt': '', 'model_type': 'vicuna7b', 'use_lora': False, 'max_txt_len': 550}, 'preprocess': {'vis_processor': {'train': {'name': 'blip2_image_train', 'image_size': 224}, 'eval': {'name': 'blip_image_eval', 'image_size': 224}}, 'text_processor': {'train': {'name': 'blip_caption'}, 'eval': {'name': 'blip_caption'}}}, 'datasets': {'ooc': {'data_type': 'images', 'build_info': {'annotations': {'train': {'storage': 'train.json'}, 'val': {'storage': 'val.json'}, 'test': {'storage': 'test.json'}}, 'images': {'storage': 'images/'}, 'vis_entity': {'storage': 'visentity.json', 'use_visentity': False}}, 'vis_processor': {'train': {'name': 'blip2_image_train', 'image_size': 224}, 'eval': {'name': 'blip_image_eval', 'image_size': 224}}, 'text_processor': {'train': {'name': 'blip_caption_revised', 'max_words': 256}, 'eval': {'name': 'blip_caption_revised', 'max_words': 256}}}}}, 'args': Namespace(cfg_path='lavis/projects/instructblip2/train/ooc_ft.yaml', options=None, wname=None, use_lora=False), '_wandb': {}}
2025-02-24 08:12:39,646 INFO    MainThread:42048 [wandb_init.py:init():784] starting backend
2025-02-24 08:12:39,646 INFO    MainThread:42048 [wandb_init.py:init():788] sending inform_init request
2025-02-24 08:12:39,681 INFO    MainThread:42048 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2025-02-24 08:12:39,681 INFO    MainThread:42048 [wandb_init.py:init():803] backend started and connected
2025-02-24 08:12:39,686 INFO    MainThread:42048 [wandb_init.py:init():896] updated telemetry
2025-02-24 08:12:39,712 INFO    MainThread:42048 [wandb_init.py:init():920] communicating run to backend with 90.0 second timeout
2025-02-24 08:12:39,821 INFO    MainThread:42048 [wandb_init.py:init():995] starting run threads in backend
2025-02-24 08:12:39,972 INFO    MainThread:42048 [wandb_run.py:_console_start():2377] atexit reg
2025-02-24 08:12:39,972 INFO    MainThread:42048 [wandb_run.py:_redirect():2227] redirect: wrap_raw
2025-02-24 08:12:39,973 INFO    MainThread:42048 [wandb_run.py:_redirect():2292] Wrapping output streams.
2025-02-24 08:12:39,973 INFO    MainThread:42048 [wandb_run.py:_redirect():2317] Redirects installed.
2025-02-24 08:12:39,974 INFO    MainThread:42048 [wandb_init.py:init():1037] run started, returning control to user process
2025-02-24 08:12:39,981 WARNING MsgRouterThr:42048 [router.py:message_loop():77] message_loop has been closed
Traceback (most recent call last):
  File "/opt/anaconda3/envs/lavis/lib/python3.9/site-packages/wandb/sdk/interface/router_sock.py", line 28, in _read_message
    resp = self._sock_client.read_server_response(timeout=1)
  File "/opt/anaconda3/envs/lavis/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 285, in read_server_response
    data = self._read_packet_bytes(timeout=timeout)
  File "/opt/anaconda3/envs/lavis/lib/python3.9/site-packages/wandb/sdk/lib/sock_client.py", line 270, in _read_packet_bytes
    raise SockClientClosedError
wandb.sdk.lib.sock_client.SockClientClosedError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/anaconda3/envs/lavis/lib/python3.9/site-packages/wandb/sdk/interface/router.py", line 70, in message_loop
    msg = self._read_message()
  File "/opt/anaconda3/envs/lavis/lib/python3.9/site-packages/wandb/sdk/interface/router_sock.py", line 30, in _read_message
    raise MessageRouterClosedError from e
wandb.sdk.interface.router.MessageRouterClosedError
2025-02-24 08:12:40,023 INFO    MsgRouterThr:42048 [mailbox.py:close():115] Closing mailbox, abandoning 0 handles.
